### PR TITLE
Adding photos and videos to HTML Templates

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] HTML Templates haven't got included example photos or videos.
[Solution] Adding example photos and videos to html templates.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/610

**Comment by mijin85cho** [comment]
Why the layout of the page's layout is changed to " style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;""?
Some pages got different layout compared to other pages with this PR.

Also, I'm guessing this PR is related with #609.
If so, please gather related fixes all in one PR,
to make it possible to check the update without confusion, and to reduce the count of PRs.

It would happen that several reviewers check each PR separately, 
and checking this PR results that it will cause broken images & videos,
while checking #609 results that it will add not used images.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm [hunk 1]
```diff
@@ -7,12 +7,13 @@
   <link href="../snippet.css" rel="StyleSheet" type="text/css" />
   <title>Button</title>
  </head>
- <body onload="prettyPrint()" id="content">
+ <body onload="prettyPrint()" id="content" style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;">
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: Remove:  style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;

**Comment by TomaszLukawskiSam** [comment]
> Why the layout of the page's layout is changed to " style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;""?
> Some pages got different layout compared to other pages with this PR.
> 
> Also, I'm guessing this PR is related with #609.
> If so, please gather related fixes all in one PR,
> to make it possible to check the update without confusion, and to reduce the count of PRs.
> 
> Checking this PR results that it will cause broken images & videos,
> while checking #609 results that it will add not used images.

Thank you for review. I understand you and you are right "If so, please gather related fixes all in one PR,". Next time we will do it in one PR. Do you wish to merge these PRs now?

**Comment by mijin85cho** [comment]
> > Why the layout of the page's layout is changed to " style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;""?
> > Some pages got different layout compared to other pages with this PR.
> > Also, I'm guessing this PR is related with #609.
> > If so, please gather related fixes all in one PR,
> > to make it possible to check the update without confusion, and to reduce the count of PRs.
> > Checking this PR results that it will cause broken images & videos,
> > while checking #609 results that it will add not used images.
> 
> Thank you for review. I understand you and you are right "If so, please gather related fixes all in one PR,". Next time we will do it in one PR. Do you wish to merge these PRs now?

For now, as I got that #609 and #610 are related, I'll merge #609 and #610 together when both are approved. For this time you don't need merge them.

**Comment by TomaszLukawskiSam** [comment]
> > > Why the layout of the page's layout is changed to " style="margin: 0 auto; display: flex; flex-direction: column; align-items:center; justify-content: center; max-width: 1200px;""?
> > > Some pages got different layout compared to other pages with this PR.
> > > Also, I'm guessing this PR is related with #609.
> > > If so, please gather related fixes all in one PR,
> > > to make it possible to check the update without confusion, and to reduce the count of PRs.
> > > Checking this PR results that it will cause broken images & videos,
> > > while checking #609 results that it will add not used images.
> > 
> > 
> > Thank you for review. I understand you and you are right "If so, please gather related fixes all in one PR,". Next time we will do it in one PR. Do you wish to merge these PRs now?
> 
> For now, as I got that #609 and #610 are related, I'll merge #609 and #610 together when both are approved. For this time you don't need merge them.

Thank you very much!

**Comment by TomaszLukawskiSam** [comment]
But we still have to fix this PR.

**Comment by Dominnio** [comment]
Done.

**Comment by dorlinski** [comment]
Done.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm [hunk 2]
```diff
@@ -7,12 +7,13 @@
   <link href="../snippet.css" rel="StyleSheet" type="text/css" />
   <title>Button</title>
  </head>
- <body onload="prettyPrint()" id="content">
+ <body onload="prettyPrint()" id="content"">
```
[the remark this answers is not in the document]
Comment (Dominnio) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: Fixed

[the remark this answers is not in the document]
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: Fixed

**Pull request merged**